### PR TITLE
report store write flow to pd

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@BusyJay @nolouch PTAL

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -121,6 +121,8 @@ pub struct Store<T, C: 'static> {
     region_written_bytes: LocalHistogram,
     region_written_keys: LocalHistogram,
     lock_cf_written_bytes: u64,
+
+    bytes_written: u64,
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: do we need to extract a store metric structure to save those bytes and keys fields?

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -1593,6 +1600,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
         stats.set_start_time(self.start_time.sec as u32);
 
+        // report store write flow to pd
+        let bytes_written = self.engine.get_statistics_ticker_count(TickerType::BytesWritten);
+        let delta = bytes_written - self.bytes_written;
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: can bytes_written be larger than self.bytes_written?

Comment (zhangjinpeng1987) [inline src/raftstore/store/store.rs]: `bytes_written` equal or greater than `self.bytes_written`

File: src/raftstore/store/store.rs [hunk 3]
```diff
@@ -120,7 +126,8 @@ pub struct Store<T, C: 'static> {
 
     region_written_bytes: LocalHistogram,
     region_written_keys: LocalHistogram,
-    lock_cf_written_bytes: u64,
+
+    store_stat: StoreStat,
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: need to contain region_written_bytes and keys?

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @disksing

**Comment by zhangjinpeng1987** [comment]
@hhkbp2 PTAL

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**